tajo-catalog/tajo-catalog-common/src/main/java/org/apache/tajo/catalog/AlterTableDesc.java

```diff
@@ -51,6 +54,8 @@
   @Expose
   protected KeyValueSet properties;
   @Expose
+  protected Set<String> propertyKeys;
```

Comment:
Since there is no additional descriptions on this variable, it would be better if it has a more intuitive name like ```unsetPropertyKeys```.